packages/wrangler/src/r2/helpers.ts

```diff
@@ -3,6 +3,11 @@ import { fetchResult } from "../cfetch";
 import { fetchR2Objects } from "../cfetch/internal";
 import type { HeadersInit } from "undici";
 
+/**
+ * The maximum file size we can upload using the V4 API.
+ */
+export const MAX_UPLOAD_SIZE = 300 * 1024 * 1024;
```

Comment:
Should this be in helpers? lmk if not.

Reply:
Normally I'd create a `constants.ts` file but wrangler doesn't have that convention yet - up to you

Reply:
Moved to `constants.ts`